src/DynamoPackages/PackageLoader.cs

```diff
@@ -373,12 +373,13 @@ public void LoadPackages(IEnumerable<Package> packages)
         /// <summary>
         /// This method is called when custom nodes and packages need to be reloaded if there are new package paths.
         /// </summary>
-        /// <param name="newPaths"></param>
-        /// <param name="preferences"></param>
+        /// <param name="packagePaths"></param>
+        /// <param name="nodePaths"></param>
         /// <param name="customNodeManager"></param>
-        public void LoadCustomNodesAndPackages(IEnumerable<string> newPaths, IPreferences preferences, CustomNodeManager customNodeManager)
+        public void LoadCustomNodesAndPackages(IEnumerable<string> packagePaths, IEnumerable<string> nodePaths, 
+            CustomNodeManager customNodeManager, IEnumerable<string> uninstallPackageDir = null, bool checkPathInPathManager = true)
```

Comment:
please add xml documentation for `uninstallPackageDir ` and `checkPathInPathManager ` for API users to understand how to call this API

Reply:
Also, it should be `dirs` (plural) if it can be more than one directory.